app/poros/movie.rb

```diff
@@ -0,0 +1,28 @@
+class Movie
+  attr_reader :movie_id,
+              :title,
+              :vote_avg,
+              :runtime,
+              :genres,
+              :summary,
+              :cast,
+              :review_count,
+              :reviews
+
+  def initialize(movie_data, cast_data = nil, review_data = nil)
+    @movie_id = movie_data[:id]
+    @title = movie_data[:title]
+    @vote_avg = movie_data[:vote_average]
+    @runtime = movie_data[:runtime]
+    @genres = movie_data[:genres]
+    @summary = movie_data[:overview]
+    @cast = cast_data
+    @reviews = review_data
```

Comment:
I'm wondering if we could make @cast and @reviews an array of cast and review objects